src/main/java/com/revature/models/CarTripDTO.java

```diff
@@ -0,0 +1,19 @@
+package com.revature.models;
+
+import lombok.*;
+
+@Data
+@NoArgsConstructor
+public class CarTripDTO {
```

Comment:
Is a CarTripDTO actually necessary?

Reply:
We're using it to send a response with the available seats (from the Trip model) along with Car. Mostly to avoid making multiple requests per driver on the front end

Reply:
Alright, then that's fine.